honeybee_3dm/face.py

```diff
@@ -36,63 +39,115 @@ def import_faces(rhino3dm_file, tolerance=None):
     hb_shades = []
     hb_apertures = []
     hb_doors = []
-
-    # TODO: Add an input to customize layer names
-    # A Layer dictionary with layer name : (Honeybee face_type, Class) structure
-    layer_to_hb_object = {
-        'roof': (face_types.roof_ceiling, Face),
-        'wall': (face_types.wall, Face),
-        'floor': (face_types.floor, Face),
-        'airwall': (face_types.air_boundary, Face),
-        'shade': (None, Shade),
-        'aperture': (None, Aperture),
-        'door': (None, Door)
-    }
-
-    for layer in rhino3dm_file.Layers:
-        if layer.Name not in layer_to_hb_object:
-            warnings.warn(
-                f'Only objects on layers {tuple(layer_to_hb_object.keys())} will be'
-                'imported during the process of importing faces.'
-            )
-
-    # get all the objects for valid layers
-    layers = list(layer_to_hb_object.keys())
-    objects_by_layer = filter_objects_by_layers(rhino3dm_file, layers)
-
-    for layer, rhino_objects in zip(layers, objects_by_layer):
-
-        hb_face_type, hb_face_module = layer_to_hb_object[layer]
-
-        # for each rhino geometry gathered, Converting the Rhino3dm geometry
-        # into a Ladybug Face3D objects
-        for obj in rhino_objects:
-
-            lb_faces = to_face3d(obj, tolerance=tolerance)
-            name = obj.Attributes.Name
-            for face_obj in lb_faces:
-                # TODO: Double check with Chris if this naming works for energy models.
-                # if name is assigned by user use the same name for all the sub faces
-                # otherwise generate a randome name based on the layer name.
-                obj_name = name or clean_and_id_string(layer)
-                args = [clean_string(obj_name), face_obj]
-                if hb_face_type:
-                    args.append(hb_face_type)
-                    hb_face = hb_face_module(*args)
+    config = config
+    # If config file is provided and any Rhino layer is assigned
+    if config and config['HB_layers'].values():
```

Comment:
You need to check that `HB_layers` key exist in a different line before trying to get the values here. This will raise a non explicit error if the provided dictionary is missing the key.